src/coprocessor/dag/executor/limit.rs

```diff
@@ -114,7 +114,7 @@ mod test {
         // init TableScan
         let (snapshot, start_ts) = test_store.get_snapshot();
         let store = SnapshotStore::new(snapshot, start_ts, IsolationLevel::SI, true);
-        let ts_ect = TableScanExecutor::new(&table_scan, key_ranges, store).unwrap();
+        let ts_ect = TableScanExecutor::new(&table_scan, key_ranges, store, false).unwrap();
```

Comment:
It seems we should implement `collect_output_counts` for limit with `self.src.collect_output_counts`?

Reply:
No, because we do not know the last range is fully scanned.

Reply:
So it means you only `collect_output_counts ` when the last range is fully scanned?

Reply:
No, it means for `limit`, we do not know the range is fully scanned, so the result may be wrong.